test/cql-pytest/test_keyspace.py

```diff
@@ -251,6 +251,22 @@ def test_storage_options_describe(cql, scylla_only):
 
         assert "{'type': 'S3', 'bucket': '42', 'endpoint': 'localhost'}" in desc_output
 
+# Reproducer for scylladb#14139
+def test_alter_keyspace_preserves_udt(cql):
+    cql.execute(f"CREATE KEYSPACE ks WITH REPLICATION = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}}")
```

Comment:
Please use a unique name for the keyspace, not the fixed name "ks" - if we ever run multiple tests in parallel against the same cluster, or if a test fails and leaves behind a keyspace - we don't want to see weird test failures because the keyspace "ks" already exist. For example:
```python
ks = unique_name()
cql.execute(f"CREATE KEYSPACE {ks} WITH REPLICATION ...")
```